dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -284,10 +261,8 @@ private static ChatHistory PrepareChatHistory(string text, CompleteRequestSettin
 
     private static CompletionsOptions CreateCompletionsOptions(string text, CompleteRequestSettings requestSettings)
     {
-        if (requestSettings.ResultsPerPrompt < 1 ||
-            requestSettings.ResultsPerPrompt > 128)
+        if (requestSettings.ResultsPerPrompt is < 1 or > 128)
```

Comment:
It's not clear what is 128 here. Let's put these numbers in separate constant variables with names.